Data_Collection/script_Rt_epinow_canton.R

```diff
@@ -0,0 +1,121 @@
+### Reading packages ------------------------------
+
+library(pacman)
+
+p_load(char = c("tidyverse",
+                "RPostgreSQL",
+                "EpiNow2",
+                "EpiEstim",
+                "lubridate",
+                "readxl",
+                "janitor",
+                "linelist"))
+
+# Create a SSH tunnel to connect to the database ------------------------
+# Type the following in your PowerShell
+# ssh -f epigraph@epigraphhub.org -L 5432:localhost:5432 -NC
+
+
+### Connecting to the database and transferring data ------------------------
+con <- dbConnect(
+  PostgreSQL(),
+  host = 'localhost',
```

Comment:
is it possible to get these parameters from the environment?
this is what we have for now: https://github.com/thegraphnetwork/EpiGraphHub/blob/main/.env.tpl

when you change it to get these values from these environment variables,
you can run your script on CI as well for the tests

After this line:
https://github.com/thegraphnetwork/EpiGraphHub/blob/main/Makefile#L28

Add this:

`$(DOCKER) exec -T ${SERVICE} Rscript /opt/EpiGraphHub/Data_Collection/script_Rt_epinow_canton.R`

Note: ensure to use TAB for the indentation inside Makefile, it doesn't work with SPACES

let me know if you have any question.

Reply:
@dcpcamara ?

Reply:
@xmnlab how can I get these parameters from the environment? I'm not used to this.

Reply:
good question. actually it should be added to the documentation (my fault).

I am not an R expert, so I am sharing here just something I just found:

- https://stat.ethz.ch/R-manual/R-devel/library/base/html/Sys.getenv.html

maybe @fccoelho could confirm if that works well or not.

For Python code, I created a module to handle some configuration for the environment variables:

https://github.com/thegraphnetwork/EpiGraphHub/blob/main/Data_Collection/CRON_scripts/config.py#L9-L23

Also, I am using a library to read the .env file and load it into the environment variable scope:

https://github.com/thegraphnetwork/EpiGraphHub/blob/main/Data_Collection/CRON_scripts/config.py#L7

For R, maybe you will need also a 3rd-party library, for example, there is a library called doenv:

- https://rdrr.io/cran/dotenv/

And here there is an example using dotenv library:

- https://rdrr.io/cran/dotenv/man/load_dot_env.html 

If you have any problems, feel free to reach me out and we can have a quick call to implement that together.

Reply:
@xmnlab maybe it will be better if we can meet to implement this together. I have never done this before and I don't think that I'm understanding what I'm supposed to do lol.

Reply:
no problem. I will message you to schedule a meeting tomorrow.

Reply:
@xmnlab I think that `Sys.getenv` only accesses the env variables not `.env` files, but if docker can handle the declaration of all variables from the `.env` files on boot then it may work just fine.

@dcpcamara You are the R expert here! :-) All you have to learn now are some devops tricks to feel more at home! and @xmnlab  is the best teacher! By the way @xmnlab, please invite @morteza-mahdiani  to this meeting as well, he is joining the team and wants to learn more about our devops workflow!

Reply:
Thanks for the kind words, Flavio! We have scheduled a meeting already today. @morteza-mahdiani can you please provide an email so that we can send you an invitation?